client/.golangci.yml

```diff
@@ -1,21 +1,20 @@
 # THIS FILE WAS AUTOMATICALLY GENERATED, PLEASE DO NOT EDIT.
 #
-# Generated on 2024-02-09T09:47:03Z by kres decf506.
+# Generated on 2024-04-11T11:10:37Z by kres e4c6337.
 
 # options for analysis running
 run:
   timeout: 10m
   issues-exit-code: 1
   tests: true
-  build-tags: []
-  skip-dirs: []
-  skip-dirs-use-default: true
-  skip-files: []
+  build-tags: [ ]
```

Comment:
is the change in formatting ok/intended?

Reply:
not me, it's kres :)

Reply:
I know, was wondering if it was expected from kres... But anyways, no big deal.